### ISSUE TITLE
Notifications

### ISSUE THREAD

**Issue by rowi123** [revancedapps/revancedmanager#3]
I never get a notification if an update is available. If this is a feature is doesn't work.
If this isn't a feature yet, then it's a good idea to add.

**Comment by revancedapps** [comment]
Of course, it's already part of our plan. This feature will be released in the next few days.
